src/server/dstu2/observation/controllers/observation.controller.js

```diff
@@ -69,7 +71,7 @@ module.exports.getObservationById = (profile, logger) => {
 		return service.getObservationById(req, logger, context)
 			.then((observation) => {
 				if (observation) {
-					res.send(observation);
+					res.send(new Observation(observation));
```

Comment:
This may conlfict with the PR I just merged, not sure why it's not showing here. But instead of `res.send(...)`, we should use `res.status(200).json(...)`